tests/sqlparser_snowflake.rs

```diff
@@ -2975,6 +2975,25 @@ fn test_parse_show_schemas() {
     snowflake().verified_stmt("SHOW SCHEMAS IN DATABASE STARTS WITH 'abc' LIMIT 20 FROM 'xyz'");
 }
 
+#[test]
+fn test_parse_show_objects() {
+    snowflake().verified_stmt("SHOW OBJECTS");
```

Comment:
Since this introduces a new node in the AST, can we add an assertion for one of these test cases covering that the AST looks like what is expected? i.e a test in [this syle](https://github.com/apache/datafusion-sqlparser-rs/blob/e13a1b844b176746dbc5f04987e0b97d78df88b4/tests/sqlparser_snowflake.rs#L51-L59)